Sources/Clappr/Classes/Base/Core.swift

```diff
@@ -7,7 +7,6 @@ open class Core: UIBaseObject, UIGestureRecognizerDelegate {
         }
     }
     @objc fileprivate(set) open var containers: [Container] = []
-    @objc fileprivate(set) open var mediaControl: MediaControl?
```

Comment:
It makes sense to make `mediaControl` not visible in the `Core` anymore?

We're sure that all communication will be made through events?

Reply:
Yes, the idea is to remove the couple between Core and MediaControl. With this implementation, we adopt the communication to be made through events.